src/containers/toputilizers/containers/TopUtilizersResultsContainer.js

```diff
@@ -55,6 +57,83 @@ class TopUtilizersResultsContainer extends React.Component {
     });
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (this.props.isGettingNeighbors && !nextProps.isGettingNeighbors) {
+      this.countEdgeTypesPerEntity(nextProps);
+    }
+  }
+
+  // each element in 'neighbor' is an array that contains associations that belong to search parameters.
+  // maintain a count of each association type within each neighbor object and display that on the table.
+  countEdgeTypesPerEntity = (nextProps) => {
+    const countHeaders = this.createHeadersForEdgeTypes();
+    this.props.results.forEach((result) => {
+      const entityId = result.id[0];
+      // Get List of neighbors for specific entity
+      const neighborList = nextProps.neighbors.get(entityId);
+      const neighborCount = {};
+
+      // For each edge in the list of neighbors, count occurances for each unique assoc -> neighbor
+      neighborList.forEach((edge) => {
+        const associationEntityId = edge.getIn(['associationEntitySet', 'entityTypeId']);
+        const neighborEntityId = edge.getIn(['neighborEntitySet', 'entityTypeId']);
+        const isSrc = edge.get('src');
+
+        const edgeId = this.getEdgeId(associationEntityId, neighborEntityId, isSrc);
+        const edgeFqn = `count.${edgeId}`;
+        if (neighborCount[edgeFqn] !== undefined) {
+          neighborCount[edgeFqn][0] += 1;
+        }
+        else {
+          neighborCount[edgeFqn] = [1];
+        }
+      });
+
+      // Assign neighborCount to current result element
+      Object.assign(result, neighborCount, { 'count.Headers': countHeaders });
```

Comment:
It seems like you're directly modifying `props`, which is an anti-pattern (I think). `props` should be immutable, in other words, `props` should only be changed externally, by parent components passing in new values. Any other mutations based on `props` should be applied to `state`.